### PR TITLE
Navigation update

### PR DESCRIPTION
**Pull request by volodymyr-x**

### CODE REVIEW

File: app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/ChooseLanguageFinalPage.kt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import com.example.awesomelanguagelearning.chooseLanguage.domain.entity.ChooseLanguagePageState
+import com.example.awesomelanguagelearning.core.ui.theme.AppTheme
+import com.example.awesomelanguagelearning.core.ui.views.HorizontalSpacer
+import com.example.awesomelanguagelearning.core.ui.views.TextButton
+import com.example.awesomelanguagelearning.core.ui.views.TextTitle
+
+@Composable
+fun ChooseLanguageFinalPage(
+    state: ChooseLanguagePageState,
+    onEvent: (ChooseLanguageEvent) -> Unit = {}
```
Comment (slothdeboss) [inline app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/ChooseLanguageFinalPage.kt]: @igel911 might want to pass modifier as a parameter here 👀

File: app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguageOverviewPage.kt [hunk 2]
```diff
@@ -15,21 +15,21 @@ import androidx.compose.ui.tooling.preview.Preview
...
 import com.example.awesomelanguagelearning.core.ui.views.TextButton
 import com.example.awesomelanguagelearning.core.ui.views.TextTitle
 
 @Composable
-fun ChooseLanguageLastPage(
+fun ChooseLanguageOverviewPage(
     state: ChooseLanguagePageState,
-    onNextClick: () -> Unit = {}
+    onEvent: (ChooseLanguageEvent) -> Unit = {}
 ) {
```
Comment (slothdeboss) [inline app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguageOverviewPage.kt]: @igel911 same here, let's add `modifier` here

File: app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguagePage.kt [hunk 3]
```diff
@@ -18,24 +18,22 @@ import androidx.compose.ui.unit.dp
...
 import com.example.awesomelanguagelearning.core.ui.views.TextTitle
-import java.util.UUID
 
 @Composable
 fun ChooseLanguagePage(
     state: ChooseLanguagePageState,
-    onNextClick: () -> Unit = {},
-    onItemClick: (UUID) -> Unit = {}
+    onEvent: (ChooseLanguageEvent) -> Unit = {}
 ) {
```
Comment (slothdeboss) [inline app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguagePage.kt]: @igel911 and here 👀

**Review by bohdanzdaniekin — APPROVED** [review]

File: app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/ChooseLanguageFinalPage.kt (hunk 1 above)
Comment (volodymyr-x) [reply app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/ChooseLanguageFinalPage.kt]: @slothdeboss So you're a big fan of modifiers, I see.

File: app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguageOverviewPage.kt (hunk 2 above)
Comment (volodymyr-x) [reply app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguageOverviewPage.kt]: @slothdeboss Is it a phobia or something?

File: app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguagePage.kt (hunk 3 above)
Comment (volodymyr-x) [reply app/src/main/java/com/example/awesomelanguagelearning/chooseLanguage/ui/views/ChooseLanguagePage.kt]: @slothdeboss Are those modifiers in the room with us right now?

**Review by slothdeboss — APPROVED** [review]

**Pull request merged**